test_scripts/Resumption/InteriorVehicleData/OnHashChange/001_OnHashChange_by_module_subscription_without_moduleId.lua

```diff
@@ -0,0 +1,43 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0188-get-interior-data-resumption.md
+--
+-- Description: SDL sends OnHashChange notification after successful subscription to interior vehicle data
+--  with default moduleId
```

Comment:
@GetmanetsIrina Mobile application does not send any moduleId in its request during this check. 
Please rephrase test case description.
This comment relates to other scripts of the PR too.